.travis.yml

```diff
@@ -6,6 +6,11 @@ rvm:
   - 1.9.3
   - 2.1.5
 
+before_install:
+  - gem update --system
+  - gem --version
+  - gem install bundler --pre
```

Comment:
I'm not a huge fan of these, but it's only for tests, and I don't see `rubygems` and/or `bundler` having a bad release without releasing immediately a fix.